tests/hello_nexus/helpers.py

```diff
@@ -0,0 +1,41 @@
+import temporalio.api
+import temporalio.api.common
+import temporalio.api.common.v1
+import temporalio.api.enums.v1
+import temporalio.api.nexus
+import temporalio.api.nexus.v1
+import temporalio.api.operatorservice
+import temporalio.api.operatorservice.v1
+import temporalio.nexus
+import temporalio.nexus.handler
+from temporalio.client import Client
+
+
+# TODO: copied from sdk-python tests/helpers/nexus
+async def create_nexus_endpoint(
```

Comment:
So this is fairly bad dx I think for testing Nexus from a user POV. Can we open an issue in CLI (but may need to be in server first) to allow Nexus endpoints to be created as part of `server start-dev` similar to what we do with namespaces?